PostHog vs Amplitude blog article

andyvan-ph opened the pull request:
## Changes

Draft of new PostHog vs Amplitude article. This could, in future, be a landing page but for now it's the start of a template for these kinds of pages in future.

Still draft atm, but open to comments on the basic shape of things and any technical details. All the stuff in the comparison tables needs double checking before publishing.

## Checklist
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)
- [ ] Words are spelled using American English
- [ ] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

andyvan-ph commented:
Paging @charlescook-ph @joethreepwood @jamesefhawkins @marcushyett-ph for feedback on this as it's 99% done now.

I've tried to find a balance between skimability and providing the kind of depth our end users will want. I would particularly appreciate any sense checking on the comparison tables etc.

andyvan-ph commented:
@corywatilo I'd appreciate your designer's eye over the tables in this. I based on your tweaks to Joe's one, but I feel like the columns are a little bunched up but I'm at the limit of HTML skills where tables are concerned!

marcushyett-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+### 2. It's built for engineers
+Unlike Amplitude, PostHog is built for software developers. PostHog autocaptures events, so you don't need to waste time instrumenting events every time you update your app or website. Our pricing is transparent, and we're entirely self-serve. You can setup PostHog and start paying us without ever speaking to a sales rep, but we have an [awesome customer success team](/signup/self-host/get-in-tou ...
+
+### 3. You control your data
```
I feel this should be number 1 in the list. I feel this is more generally applicable than the first one.

andyvan-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+    	</tr>
+    	<tr>
+			<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
```
Preview link: https://deploy-preview-3404--posthog.netlify.app/blog/posthog-vs-amplitude#feature-comparison

marcushyett-ph commented:
Overall looking great to me @andyvan-ph

simfish85 commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+
+Amplitude's entry-level tier is free to use up to 10 million events per month, which compares very favorably to rivals like Mixpanel (up to 100k monthly users), Pendo (up to 1,000 monthly users) and Heap (up to 10,000 sessions per month). Once you exceed 10 million events, you'll need to speak to Amplitude's sales team to proceed further.
+
+PostHog's model is slightly different. PostHog Open Source, which [anyone can self-host](/docs/self-host), is totally free for life. PostHog Cloud is free up to 1 million events per month, and this includes premium features like Experimentation and Correlation Analysis not available on Amplitude's free Starter tier.
```
I'd say PostHog Cloud and PostHog Scale are free up to 1m events here

joethreepwood commented:
Fixed the table for you.

joethreepwood reviewed (commented):
Thrown in a smattering of comments (some of which are tiny nits I'll go in and commit now), but this looks good overall. I think we could make this more visually impactful, however. Throw in some screenshots, quotes from users to help it be less wall-of-text. Also think we should be linking out to product pages and docs whenever we mention features, but defer to you on that.

andyvan-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+
+All paid plans of PostHog include our Experimentation suite for running A/B and multivariate tests, while feature flags and session recording are no additional cost features on all versions of PostHog, including [PostHog Open Source](/signup/self-host).  
+
+Amplitude analytics doesn't include feature flags or experimentation by default, instead bundling them as part of a separate product, Amplitude Experiment. Like all Amplitude products, Experiment is a price on application only product. 
```
It means you have to talk to sales to find out how much it costs

joethreepwood replied:
I think I'd just say that then. "a price on application only product" reads very weirdly to weirdly to me. Maybe it'd be clearer if it was a price-on-application only, but I'd still say that saying there pricing isn't transparent and you need to contact sales is clearer.

andyvan-ph replied:
Kind of see your point. Have changed in the latest revision.

andyvan-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>HIPAA compliance</td>
```
I mean, we do claim this already afaik @charlescook-ph?

charlescook-ph replied:
I think we can say yes because other solutions don't actually enable you to take that step, whereas we do (so I'm happy for us to claim we are HIPAA compliant)

andyvan-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,420 @@
+
+Amplitude analytics doesn't include feature flags or experimentation by default, instead bundling them as part of a separate product, Amplitude Experiment. Like all Amplitude products, Experiment is a price on application only product. 
+
+### Transparent pricing
```
I don't totally follow

andyvan-ph replied:
Got you. Have updated now.

andyvan-ph commented:
I've made some revisions based on feedback and my own changes.

On the tables, I've changed it so it's comparing PostHog Scale as opposed to 'PostHog Self-host' as it felt a abit abstract. I added a 'Host yourself' row to highlight the difference betweem it and Cloud etc.

andyvan-ph commented on contents/blog/posthog-vs-amplitude.md:
```diff
@@ -0,0 +1,412 @@
+
+All versions of PostHog are transparently priced – even the [Enterprise version](/pricing), which adds single sign-on (SAML), team training, instance monitoring, and more. Paid tiers (excluding Enterprise) are free up to 1 million events per month, and pricing calculators show exactly how much you'll pay when you go over 1 million. The more you use, the cheaper it gets per event.
+
+Amplitude doesn't share its pricing publibly, so you have to speak to their sales team to get up and running, and negotiate a price based on your usage. It's a bit like a really expensive restaurants. It looks fancy and expensive, but you only find out how expensive it is when you're already sat at your table.
```
```suggestion
Amplitude doesn't share its pricing publibly, so you have to speak to their sales team to get up and running, and negotiate a price based on your usage. It's a bit like a really expensive restaurant. It looks fancy and expensive from the outside, but you only find out just how expensive it is when you're already sat at your table.
```
